Dockerfile.release

```diff
@@ -1,11 +1,9 @@
-FROM alpine:3.9
+FROM alpine:3.12
 
 ARG BINARY=nri-prometheus
 
 RUN apk add --no-cache --upgrade \
-        ca-certificates \
-        musl=1.1.20-r5
```

Comment:
It is unclear to me why this specific version of musl was pinned before. I did some basic sanity checks (i.e. checking that the integration launches) without it and it seems to run properly.

Reply:
@carlosroman According to the blame it was you who introduced this pin, do you by chance remember the reason? Thanks!

Reply:
according to this list of packages https://pkgs.alpinelinux.org/packages?name=musl&branch=v3.12&repo=main&arch=x86_64. Alpine 3.12 has a more recent version of musl, so we should be fine removing that pinned version (unless the pinned version was for some backwards compat issue)